templates/admin/form.html.twig

```diff
@@ -0,0 +1,133 @@
+{# @see https://symfony.com/bundles/EasyAdminBundle/current/design.html#form-field-templates #}
+{% block _Theme_themeCategories_widget %}
+    {% set id = 'form-theme-themeCategories' %}
+    <table class="table table-striped table-ea-collection" id="{{ id }}">
```

Comment:
```suggestion
    <table class="table table-striped table-ea-collection" id="{{ id }}">
    <caption>a little something about this table</caption>
```

Reply:
I appreciate this suggestion, but I don't know how to caption this table since it's used only for an alternative visual representation of data items (rows). Accessibility-wise this is not good, but I don't know how to make it better – I'm just the upgrade programmer here …